spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/context/MessageSourceProperties.java

```diff
@@ -71,6 +73,12 @@
 	 */
 	private boolean useCodeAsDefaultMessage = false;
 
+	/**
+	 * Whether to use a {@link ReloadableResourceBundleMessageSource} instead of the
```

Comment:
Configuration key descriptions can't have javadoc tag (see documentation). I think we need to improve the description as high-level descriptions shouldn't contain implementation details (rather it should describe what the feature does).